gcalcli/gcalcli.py

```diff
@@ -1370,9 +1370,10 @@ def Remind(self, minutes=10, command=None, use_reminders=False):
                     # don't remind if all reminders haven't arrived yet
                     continue
 
-            if self.options['military']:
+            try:
```

Comment:
I'd prefer to fix this with:

```
if self.options.get('military'):
```

Because `.get` returns `None` when the key doesn't exist, there's no KeyError raised, and `None` is falsey so we go into the `else` branch.

Also, would you mind fixing the camelCasing in this block when you resubmit :)